ToggleItem/ToggleItem.js

```diff
@@ -135,7 +135,7 @@ const defaultConfig = {
  *
  * @class ToggleItemDecorator
  * @memberof moonstone/ToggleItem
- * @mixes ui/ToggleItem.ToggleItemDecorator
+ * @mixes moonstone/ui/ToggleItem.ToggleItemDecorator
```

Comment:
Is this the right one? I don't see `ui/ToggleItemDecoroator`

Reply:
Yes,. ToggleItemDecorator is in the ui/ToggleItem/ToggleItem.js file